docset/winserver2012-ps/scheduledtasks/New-ScheduledTask.md

```diff
@@ -163,13 +163,14 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Trigger
 ### -Trigger
 Specifies an array of one or more trigger objects that cause a scheduled task to start.
 
 A trigger is a set of criteria that starts a scheduled task when the criteria are met.
 You can use a time-based trigger or an event-based trigger to start a task, and one or more triggers can start a task.
 A task can have up to 48 triggers.
-For more information about triggers, see Triggershttp://technet.microsoft.com/en-us/library/cc748841.aspx.
+For more information about triggers, see [Triggers](https://technet.microsoft.com/en-us/library/cc748841.aspx).
```

Comment:
```suggestion
For more information about triggers, see [Triggers](https://technet.microsoft.com/library/cc748841.aspx).
```

Reply:
Good catch, I got the http but missed en-us :)